backend/service/chaos/experimentation/experimentstore/experiment.go

```diff
@@ -13,7 +15,7 @@ type Experiment struct {
 	Config *ExperimentConfig
 }
 
-func (rc *Experiment) toProto() (*experimentationv1.Experiment, error) {
+func (rc Experiment) Proto(now time.Time) (*experimentationv1.Experiment, error) {
```

Comment:
Feels a bit weird to have a conversion to proto take a current time? Maybe rename this to make it clear that it evaluates the state based on current time?

Reply:
Not a huge fan either but methods that convert to proto are usually called `Proto` or `toProto` so I think it's fine to follow this convention here. I know that our method is not a standard proto conversion but I think it should be fine.

Reply:
mostly for testing purposes